modules/cli/src/main/java/org/apache/ignite/internal/cli/call/connect/ConnectCall.java

```diff
@@ -81,8 +86,8 @@ public CallOutput<String> execute(UrlCallInput input) {
             stateConfigProvider.get().setProperty(CliConfigKeys.LAST_CONNECTED_URL.value(), nodeUrl);
 
             String jdbcUrl = jdbcUrlFactory.constructJdbcUrl(configuration, nodeUrl);
-            session.connect(new SessionInfo(nodeUrl, fetchNodeName(nodeUrl), jdbcUrl, username));
-
+            session.connect(new SessionInfo(nodeUrl, fetchNodeName(nodeUrl), jdbcUrl, username, ConnectionStatus.OPEN));
+            connectionHeartBeat.start(session, clientFactory);
```

Comment:
We have `AsyncSessionEventListener`, I think you can use it for connect/disconnect logic.